### PR TITLE
Context

### PR DESCRIPTION
**Pull request by omgaikwad**
added ProductContext

### CODE REVIEW

File: src/components/HorizontalVideoCard/HorizontalVideoCard.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.HorizontalVideoCard {
+  width: 402px;
+  height: 98px;
+  margin-bottom: 2rem;
+  /* border: 1px solid var(--spotlight-primary); */
```
Comment (Hamdrive) [inline src/components/HorizontalVideoCard/HorizontalVideoCard.css]: Take care of pushing commented code

File: src/pages/Explore/Explore.jsx [hunk 2]
```diff
@@ -3,8 +3,14 @@ import "./Explore.css";
 import Navbar from "../../components/Navbar/Navbar";
 import Sidebar from "../../components/Sidebar/Sidebar";
 import VideoCard from "../../components/VideoCard/VideoCard";
+import { useVideoContext } from "../../context/video-context";
 
 const Explore = () => {
+  const { videoList } = useVideoContext();
+  console.log(videoList);
```
Comment (Hamdrive) [inline src/pages/Explore/Explore.jsx]: Take care to cleanup consoles before pushing for deployment

**Review by Hamdrive — COMMENTED** [review]
Good code written, liked the use of components wherever possible!
Overall good practices followed

File: src/backend/db/videos.js [hunk 3]
```diff
@@ -1,21 +1,70 @@
+import { v4 as uuid } from "uuid";
+
 /**
```
Comment (hinanshisuthar) [inline src/backend/db/videos.js]: You could clean up the commented code

**Review by hinanshisuthar — APPROVED** [review]
Gone through the code.
- Liked how you made reuse of components
- Overall code structure is good
Good to go from my side nice work done, Om!

File: src/components/HorizontalVideoCard/HorizontalVideoCard.css (hunk 1 above)
Comment (omgaikwad) [reply src/components/HorizontalVideoCard/HorizontalVideoCard.css]: I'll remove it!

File: src/pages/Explore/Explore.jsx (hunk 2 above)
Comment (omgaikwad) [reply src/pages/Explore/Explore.jsx]: Sure!

File: src/backend/db/videos.js (hunk 3 above)
Comment (omgaikwad) [reply src/backend/db/videos.js]: Yes, I'll clean it up!

**Pull request merged**